app/assets/images/payment_icons/alchemy.svg

```diff
@@ -0,0 +1 @@
AAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAA AAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAA okRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAA AAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAA AAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQF AAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAA AAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACA KNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAA AAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAA AAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0B AAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAA AAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAg SjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAA AAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAA AACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMA AAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAA AAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACI Ek0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAA AAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAA AAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQA AAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAA AAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACi RFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAA AAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAA AACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUA AAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAA AAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo 0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAA AAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAA AACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEA AAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAA AAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBK NAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAA AAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAA AIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAA AAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAA AAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAACja drVi+4q57eTdAAAAAElFTkSuQmCC" id="image883" clip-path="url(#clipPath889)" transform="matrix(.02667 0 0 .02667 49.861 13.826)" opacity="1" stop-opacity="1"/></svg>
```

Comment:
There a few elements that I don't see used.

- `<path id="path836">`
- `<path id="rect866">`
- `<clipPath id="clipPath864">`
- `<path id="rect891">`

Can we also remove `id="svg840"`?

In summary, can we please remove anything that is not used?